src/js/index.js

```diff
@@ -16,17 +16,16 @@ model.init();
 var ui = require('./components');
 
 var g = model.Scene.child('marks.group'),
-    g2 = g.child('marks.group'),
-    g3 = g2.child('marks.group'),
+    g2 = model.Scene.child('marks.group'),
     p = model.pipeline('cars'),
     p2 = model.pipeline('jobs'),
     p3 = model.pipeline('gapminder');
 
 // Pre-populate state with one rect, one symbol, one text & one line mark
-g3.child('marks.rect');
-g.child('marks.symbol');
+g2.child('marks.rect');
+g2.child('marks.symbol');
 g2.child('marks.line');
-g.child('marks.text');
+g2.child('marks.text');
```

Comment:
Totally cool w/ these changes but it does remove the ability to test nested groups, I defer to @arvind on how important that is for regression-catching purposes. Not a blocker to this PR.